pkg/mco/mcplist.go

```diff
@@ -12,10 +12,19 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-func ListMCP(apiClient *clients.Settings, listOptions metav1.ListOptions) ([]*MCPBuilder, error) {
-	glog.V(100).Infof("Listing all MCP resources with the options %v", listOptions)
+func ListMCP(apiClient *clients.Settings, listOptions ...metav1.ListOptions) ([]*MCPBuilder, error) {
```

Comment:
Can you change the parameter name to options so it is consistent with the other list functions?